dan_layer/engine/src/runtime/impl.rs

```diff
@@ -765,6 +766,25 @@ impl RuntimeInterface for RuntimeInterfaceImpl {
         Ok(())
     }
 
+    fn create_free_test_coins(&self, amount: u64, private_key: RistrettoSecretKey) -> Result<(), RuntimeError> {
+        let commitment = get_commitment_factory().commit(&private_key, &RistrettoSecretKey::from(amount));
+        let resource = ResourceContainer::confidential(
+            CONFIDENTIAL_TARI_RESOURCE_ADDRESS,
+            Some((commitment.as_public_key().clone(), ConfidentialOutput {
+                commitment,
+                stealth_public_nonce: None,
+                encrypted_value: None,
+                minimum_value_promise: 0,
+            })),
+            Amount::new(amount as i64),
```

Comment:
This creates a confidential asset with `amount` blinded AND `amount` revealed (2 x `amount` in funds created). It's probably easier to create the revealed funds only and no commitment since revealed funds are used for fees.

Reply:
I don't think it's terrible right now, this is mainly for testing